References/abbreviations.md

```diff
@@ -36,7 +36,9 @@
 
 *[OOB Management]: Out-of-band Management - A remote, hardware-based management technology that allows administrators to perform actions on network assets or devices using a secure alternative to LAN-based communication protocols regardless of OS state.
 
-*[Open AMT Cloud Toolkit]: Open Active Management Technology Cloud Toolkit - open source modular microservices and libraries for integration of Intel® Active Management Technology.
+*[DMT]: Device Management Toolkit - Provides open source modular microservices, applications and libraries for integration of Intel® Active Management Technology.
```

Comment:
[DMT]: Device Management Toolkit - Provides open source modular services for integrating Intel® Active Management Technology. Previously known as Open Active Management Technology (AMT) Cloud Toolkit.  Can we reduce the verbose?

Reply:
DMT is not approved by T&B (not to mention the drug reference), just like OACT is not.  [Device Management Toolkit] is what is approved.